doc/dev/development.md

```diff
@@ -7,24 +7,27 @@ After which here are other Mixer specific docs you should look at:
 
 - [Writing Mixer adapters](./adapters.md)
 
-## Using Mixer
-
-You will need to edit
-[`testdata/configroot/scopes/global/adapters.yml`](../../testdata/configroot/scopes/global/adapters.yml)
-to setup kubeconfig per the instructions at the end of the file
-
-After you've done the above and once you've built the source tree, you can run Mixer in a basic mode using:
+## Using Mixer locally
+The following command runs Mixer locally using local configuration.
+The default configuration contain adapters like `stackdriver` that connect to outside systems. If you do not intend to use the  adapter for local testing, you should move `testdata/config/stackdriver.yaml` out of the config directory, otherwise you will see repeated logging of configuration errors.
 
 ```shell
-./bazel-bin/cmd/server/mixs server \
-  --configStoreURL=fs://$(pwd)/testdata/configroot \
-  --alsologtostderr
+KUBECONFIG=${HOME}/.kube/config bazel-bin/cmd/server/mixs server --logtostderr --configStore2URL=fs://$(pwd)/testdata/config --configStoreURL=fs://$(pwd)/testdata/configroot  -v=4
 ```
 
 You can also run a simple client to interact with the server:
 
+The following command sends a `check` request to Mixer.
 ```shell
-./bazel-bin/cmd/client/mixc check -a target.service=f.default.svc.cluster.local \
-  --string_attributes source.uid=kubernetes://xyz.default
-Check RPC returned OK
+bazel-bin/cmd/client/mixc check  -v 2 --string_attributes destination.service=abc.ns.svc.cluster.local,service.name=myservice   --stringmap_attributes request.headers=clnt2:abc,destination.labels=app:ratings,source.labels=version:v2
```

Comment:
`service.name` isn't a well-known attribute.  did you mean to send something else (like `destination.name` or `source.name`)?

I'd like to see a full set of attributes sent in one of these commands, including ip_addresses.